src/tls/tls.c

```diff
@@ -788,11 +847,19 @@ static qd_error_t _update_tls_config(qd_tls_config_t *tls_config, const qd_ssl2_
     old_cfg = tls_config->proton_tls_cfg;
```

Comment:
Line up all the `=`

Reply:
@ganeshmurthy I think the latest push resolves all outstanding issues: please re-review thanks.